### PR TITLE
Remove Settings from Twitch/SoundCloud

### PR DESCRIPTION
**Pull request by Alanexei**

### CODE REVIEW

File: websites/S/SoundCloud/presence.ts [hunk 1]
```diff
@@ -134,9 +134,9 @@ const statics = {
 presence.on("UpdateData", async () => {
   const path = location.pathname.replace(/\/?$/, "/");
 
-  const showBrowsing = await presence.getSetting("browse");
-  const showSong = await presence.getSetting("song");
-  const showTimestamps = await presence.getSetting("timestamp");
+  const showBrowsing = true;
```
Comment (SeMiD) [inline websites/S/SoundCloud/presence.ts:137]: **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)

File: websites/S/SoundCloud/presence.ts [hunk 2]
```diff
@@ -134,9 +134,9 @@ const statics = {
...
+  const showSong = true;
```
Comment (SeMiD) [inline websites/S/SoundCloud/presence.ts:138]: **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)

File: websites/S/SoundCloud/presence.ts [hunk 3]
```diff
@@ -134,9 +134,9 @@ const statics = {
...
+  const showTimestamps = true;
```
Comment (SeMiD) [inline websites/S/SoundCloud/presence.ts:139]: **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)

File: websites/S/SoundCloud/presence.ts [hunk 4]
```diff
@@ -156,7 +156,7 @@ presence.on("UpdateData", async () => {
   const playButton = document.querySelector(".playControls__play.playing");
   const playing = playButton ? true : false;
```
Comment (SeMiD) [inline websites/S/SoundCloud/presence.ts:157]: **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)

File: websites/T/Twitch/presence.ts [hunk 5]
```diff
@@ -102,10 +102,10 @@ const statics = {
 presence.on("UpdateData", async () => {
   const path = location.pathname.replace(/\/?$/, "/");
 
-  const showBrowsing = await presence.getSetting("browse");
-  const showLive = await presence.getSetting("live");
-  const showVideo = await presence.getSetting("video");
-  const showTimestamps = await presence.getSetting("timestamp");
+  const showBrowsing = true;
```
Comment (SeMiD) [inline websites/T/Twitch/presence.ts:105]: **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)

File: websites/T/Twitch/presence.ts [hunk 6]
```diff
@@ -102,10 +102,10 @@ const statics = {
...
+  const showLive = true;
```
Comment (SeMiD) [inline websites/T/Twitch/presence.ts:106]: **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)

File: websites/T/Twitch/presence.ts [hunk 7]
```diff
@@ -102,10 +102,10 @@ const statics = {
...
+  const showVideo = true;
```
Comment (SeMiD) [inline websites/T/Twitch/presence.ts:107]: **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)

File: websites/T/Twitch/presence.ts [hunk 8]
```diff
@@ -102,10 +102,10 @@ const statics = {
...
+  const showTimestamps = true;
```
Comment (SeMiD) [inline websites/T/Twitch/presence.ts:108]: **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)

File: websites/S/SoundCloud/presence.ts [hunk 9]
```diff
@@ -134,9 +134,9 @@ const statics = {
 presence.on("UpdateData", async () => {
   const path = location.pathname.replace(/\/?$/, "/");
 
-  const showBrowsing = await presence.getSetting("browse");
-  const showSong = await presence.getSetting("song");
-  const showTimestamps = await presence.getSetting("timestamp");
+  const showBrowsing = true,
```
Comment (SeMiD) [inline websites/S/SoundCloud/presence.ts:137]: **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)

File: websites/T/Twitch/presence.ts (hunk 5 above)
Comment (SeMiD) [inline websites/T/Twitch/presence.ts]: **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)

File: websites/T/Twitch/presence.ts (hunk 6 above)
Comment (SeMiD) [inline websites/T/Twitch/presence.ts]: **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)

File: websites/T/Twitch/presence.ts (hunk 7 above)
Comment (SeMiD) [inline websites/T/Twitch/presence.ts]: **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)

File: websites/T/Twitch/presence.ts (hunk 8 above)
Comment (SeMiD) [inline websites/T/Twitch/presence.ts]: **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)

File: websites/T/Twitch/presence.ts [hunk 10]
```diff
@@ -102,10 +102,10 @@ const statics = {
 presence.on("UpdateData", async () => {
   const path = location.pathname.replace(/\/?$/, "/");
 
-  const showBrowsing = await presence.getSetting("browse");
-  const showLive = await presence.getSetting("live");
-  const showVideo = await presence.getSetting("video");
-  const showTimestamps = await presence.getSetting("timestamp");
+  const showBrowsing = true,
```
Comment (SeMiD) [inline websites/T/Twitch/presence.ts:105]: **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)

**Comment by Alanexei** [comment]
Yeah, nevermind. Too many check errors.

**Pull request closed without merging**